develop-docs/self-hosted/troubleshooting/index.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Self-Hosted Troubleshooting
+sidebar_title: Troubleshooting
+sidebar_order: 9000
+---
+
+Please keep in mind that the [self-hosted repository](https://github.com/getsentry/self-hosted) is geared towards low traffic loads (less than ~1 million submitted Sentry events per month). Folks needing larger setups or having event spikes can expand from here based on their specific needs and environments. If this is not your cup of tea, you are always welcome to [try out hosted Sentry](https://sentry.io/signup/).
```

Comment:
```suggestion
Please keep in mind that the [self-hosted repository](https://github.com/getsentry/self-hosted) is geared towards low traffic loads (*less than ~1 million submitted Sentry events per month*). Folks needing larger setups or having event spikes can expand from here based on their specific needs and environments. If this is not your cup of tea, you are always welcome to [try out hosted Sentry](https://sentry.io/signup/).
```

Reply:
I don't think this needs to be emphasized.

Reply:
Alrighty :)